src/utils/UrqlCombinedError.re

```diff
@@ -34,12 +34,12 @@ type positions = array(int);
 type originalError = Js.Exn.t;
 
 /* Extension fields to add to the formatted error. */
-type extensions('a) = Js.t({..} as 'a);
+type extension;
```

Comment:
Decided to make this type abstract. Basically, an index signature like this can be implemented in BuckleScript using `Js.Dict.t`, but the type of `extension` is `any` in the official `graphql` bindings, so it seems to make sense to leave it abstract here.

Reply:
That seems like the right choice; It's impossible to know what a user might stick in here. I'd say the only "safe" choice would be a `Js.Json`, which implies to a user of these bindings that `bs-json` et al can be used to decode the extensions